Library/Formula/yelp-tools.rb

```diff
@@ -0,0 +1,19 @@
+class YelpTools < Formula
+  homepage "https://github.com/GNOME/yelp-tools"
+  url "http://ftp.gnome.org/pub/gnome/sources/yelp-tools/3.16/yelp-tools-3.16.1.tar.xz"
```

Comment:
Please use this `url`:
https://download.gnome.org/sources/yelp-tools/3.16/yelp-tools-3.16.1.tar.xz